public/sitemap.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If it is in the postbuild command, you might want to add this file to `.gitignore`